ArcGISRuntimeSDKQt_CppSamples/Geometry/CreateAndEditGeometries/CreateAndEditGeometries.cpp

```diff
@@ -216,6 +256,17 @@ void CreateAndEditGeometries::deleteSelectedElement()
   emit canUndoOrRedoChanged();
 }
 
+void CreateAndEditGeometries::setUniformScaleMode(bool isUniformScale)
```

Comment:
I would consider renaming this to `setScaleMode`. Uniform is one option.

Also I would make a new enum, or export `GeometryEditorScaleMode` into QML so you can use it directly.

Reply:
Good idea, I'll make those changes